config/environments/development.rb

```diff
@@ -1,4 +1,6 @@
-require "active_support/core_ext/integer/time"
+# frozen_string_literal: true
```

Comment:
What do frozen_string_literal stand for? No need to remove it, just a question.

Reply:
It is not my wish, Rubocop did it) So it makes all strings frozen - I can't call methods on strings which modifes them. I don't know why it's necessary, there must be something in Ruby Style Guides

Reply:
https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do